src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -122,7 +125,7 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
   }
 
   buildIdPrefix(field: CaseField): string {
-    return this.isCompoundPipe.transform(field) ? `${this.idPrefix}${field.id}_` : `${this.idPrefix}`;
+    return this.isCompoundPipe.transform(field) ? `${this.idPrefix}_` : `${this.idPrefix}`;
```

Comment:
👍

Reply:
I wonder why did we have the change in the first place? 😄
Talking about `${this.idPrefix}${field.id}_`

Reply:
it is because for complex types idPrefix had duplicates, for example interimReturns_addressAttended_addressAttended_PostCode
As well I couldn't find any usage of idPrefix in the current codebase! Neither in management_web